overlord/ifacestate/ifacemgr.go

```diff
@@ -81,6 +81,7 @@ func Manager(s *state.State, hookManager *hookstate.HookManager, runner *state.T
 	addHandler("auto-connect", m.doAutoConnect, m.undoAutoConnect)
 	addHandler("gadget-connect", m.doGadgetConnect, nil)
 	addHandler("auto-disconnect", m.doAutoDisconnect, nil)
+	addHandler("hotplug-update-slot", m.doHotplugUpdateSlot, nil)
```

Comment:
should this one have an undo part?  not entirely sure myself ATM.  right now what happens, if in the  disconnect, update, (re)connect change we error on reconnect?

Reply:
Hmm, undo seems a bit dubious here (maybe we should discuss it). This task is basically about "I see device I know and have a slot for it already, but some of its attributes affecting the slot have changed (such as device path)". The device is inserted and at a new path, so I'm not sure how to undo such change and what it means?